frontend/test/app/components/ExperimentCardList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
nit: Been using a `-test.js` suffix on these files so they have something to distinguish them in text editors besides the path that often gets hidden. Seems to be the thing lots of mocha & enzyme examples do